firedrake/randomfunctiongen.py

```diff
@@ -0,0 +1,251 @@
+import numpy as np
+import inspect
+import randomgen
+
+from firedrake.function import Function
+from ufl import FunctionSpace
+
+
+"""
+
+This module wraps 'randomgen' and enables
```

Comment:
This needs to be intersphinxed so that `randomgen` is a link to their documentation.